doc/Command-Reference.md

```diff
@@ -3132,6 +3134,31 @@ Example of the module supporting target mode
   Target Mode set to 1
   ```
 
+## CMIS debug
+
+### CMIS debug loopback
+
+This command is the standard CMIS diagnostic control used for troubleshooting link and performance issues between the host switch and transceiver module.
+
+**sfputil debug loopback**
+
+- Usage:
+  ```
+  sfputil debug loopback [OPTIONS] PORT_NAME LOOPBACK_MODE
+
+  Set the loopback mode
+  host-side-input: host side input loopback mode
+  host-side-output: host side output loopback mode
+  media-side-input: media side input loopback mode
+  media-side-output: media side output loopback mode
+  none: disable loopback mode
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sfputil firmware target Ethernet88 host-side-input
```

Comment:
```suggestion
  admin@sonic:~$ sfputil debug loopback Ethernet88 host-side-input
```

Reply:
oops! I copied it from other example, let me fix it!